index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+<body>
+    
+    <!-- navbar section -->
+<nav class="navbar">
+    <h3 id="logo">Dela McRaph</h3>
+    <ul>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+    </ul>
+
+</nav>
+
+<!-- self introduction -->
+<div class="container">
+     <div class="intro"> 
+        <h4 class="primary-text">
+            I AM RAPHAEL. <br>
+            GLAD TO SEE YOU!
+        </h4>
+        <P class="primary-text-details">
+            I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! 
+            if you like what you see and have a project you need coded, don't hesitate 
+            to contact me.
+        </P>
+        <p class="lets-connect">
+            Lets Connect
+        </p>
+        <!-- social media links -->
+        <div class="social-media">
+            <a href=" " id="twitter"><img src="images/twitter.png" alt="" srcset=""> </a>
+            <a href=" " id="linkin"><img src="images/Linkedin icon.png" alt="" srcset=""> </a>
+            <a href=" " id="medium"><img src="images/medium.png" alt="" srcset=""> </a>
+            <a href=" " id="github"><img src="images/github.png" alt="" srcset=""> </a>
+            <a href=" " id="angellist"><img src="images/angellist.png" alt="" srcset=""> </a>
+    
+        </div>

```

Comment:
- [ ] Please follow the best practices for `HTML/CSS` as per the project requirement. You must need to adhere `HTML` semantics. please use `ul` and `li` tags in your work where ever you want to add the list of icons or content. Please also add the `alt` text, because it will really helpful in case the image fails to load properly, because with the help of `alt` text a user no matter who is going through your website. (It really help for those people who are visually impaired and use `screen-reader` to hear what is on the website).

**NOTE: Always think about the user first, always try to ask yourself how you can make people life easier with the help of technology**